_data/CONTRIBUTORS.yaml

```diff
@@ -610,3 +610,36 @@ Adriano Fonzino:
     git: F0nz0
     orcid: 0009-0009-3325-8554
     affiliation: University of Bari
+Tim Beck:
+    git: tb143
+    email: tim.beck@nottingham.ac.uk
+    orcid: 0000-0002-0292-7972
+    affiliation: University of Nottingham
+Marcus Buchwald:
```

Comment:
Most new contributors do not have ORCID.

Reply:
Yes thats the way it's going to be for now.  They don't have time to chase the authors and the page is needed live asap!